services/velero/7.1.5/defaults/cm.yaml

```
@@ -26,6 +26,7 @@ data:
             s3Url: http://rook-ceph-rgw-dkp-object-store.${releaseNamespace}.svc:80/
             s3ForcePathStyle: true
             insecureSkipTLSVerify: "true"
+            checksumAlgorithm: "SHA256"
```

Can we run kommander tests with no checksum algo?
```suggestion
            checksumAlgorithm: ""
```